app/i18n/en/gen.php

```diff
@@ -18,6 +18,8 @@
 		'back_to_rss_feeds' => '← Go back to your RSS feeds',
 		'cancel' => 'Cancel',
 		'create' => 'Create',
+		'delete_all_feeds' => 'Delete all feeds',
+		'delete_errored_feeds' => 'Delete feeds in error',
```

Comment:
```suggestion
		'delete_errored_feeds' => 'Delete feeds with errors',
```

Reply:
I hesitated a bit. "Errored feeds" is likely a bit too nerdy. A feed "with errors" is not necessarily the same as a feed in an error state

Reply:
More feedback / argumentations welcome

Reply:
In (an) error state?

Reply:
"Feeds with errors" 
![grafik](https://github.com/user-attachments/assets/4a2da9b0-f077-48a2-844a-85f8e34a0220)

![grafik](https://github.com/user-attachments/assets/22f6c43b-1ead-4a46-94b0-460f10af60ab)

But it is not consistent used:
![grafik](https://github.com/user-attachments/assets/73e8e725-3970-4b58-98ae-97ade348c77c)

![grafik](https://github.com/user-attachments/assets/06022805-ace5-48ce-8197-f3f16cc9c557)

And to be honest: I do not like "errors" and "problems" naming. What about "broken" or "stopped/stopped working"? Has "warnings" or an "issue"?

Reply:
*broken* would also be fine for me

Reply:
I don't see any issue with "has encountered a problem," but I'd prefer *stopped working* over *broken*, since *broken* puts the onus very much on the feed itself which is not a given.

Reply:
For a button, I would rather avoid a full sentence and keep it very short. Let's use "feeds with errors" in this PR as this is what we already have in other messages. I hope it is OK.
I will make another PR for changing the *always reachable*, which looks like a wrong French to English translation

Reply:
https://github.com/FreshRSS/FreshRSS/pull/7049